### ISSUE TITLE
Add "create" command to create new project directory

### ISSUE THREAD

**Issue by skanehira** [motemen/ghq#190]
This is a proposal.

# About this proposal.
I using ghq to manage existed repositories.
Sometimes, I want to create new repository using `mkdir` but It takes a little work.
I think when people create new repo using "create" command with ghq that will be more useful.

# About interface
People can create new project with `ghq create project`.
This command is like the `mkdir -p (ghq root)/project && cd $_ && git init`

**Comment by Songmu** [comment]
Thank you for registering the issue.

I understand the problem well But, there is a little doubt whether ghq should do it.

If the following three commands behave in the same, It may make sense to add that functionality to ghq.

```
% ghq create myproj
% ghq create username / myproj
% ghq create github.com/username/myproj
```

**Comment by skanehira** [comment]
@Songmu Thank you.
>I understand the problem well But, there is a little doubt whether ghq should do it.
I thought it too but I.
ghq is repo manager, I think the "manager" is include "create" feature.

>If the following three commands behave in the same, It may make sense to add that functionality to ghq.

agreed :)

**Comment by mattn** [comment]
I think "ghq expand" that expand full-name from the abbreviation should be useful for this.

```
$ cd $GOPATH/src/github.com/mattn/go-runewidth
$ ghq expand
github.com/mattn/go-runewidth

$ go get $(ghq expand mattn/go-something)
```

And this can be used with hub command
```
$ hub create $(ghq expand mattn/go-sugoi-package)
```

**Comment by mattn** [comment]
Ah, sorry. My design is bad since this is related to github.com .

**Comment by Songmu** [comment]
implemented at #229 and released as v0.17.0.
Please try it. Thanks!
